concepts/maps/about.md

```diff
@@ -46,24 +46,26 @@ or
 >
 > Therefore it's important to initialize a map before using it
 
-Here are some operations that you can use with map
+Here are some operations that you can do with a `map`
```

Comment:
Could you please remove the backticks around map everywhere?
The text does not refer to the keyword `map` (where we would use the backticks) but to the English word/general concept "map". That they are both the same word is just coincidence. You wouldn't write "you can do that with a `slice` for example".

Reply:
Sorry about that. Fixed!

Reply:
Thanks for the quick fix!

Reply:
Thanks for the quick approvals :D